File: Cálculo_Simples.py
id, un, valor = input().split()
id2, un2, valor2 = input().split()
un = int(un)
valor = float(valor)

un2 = int(un2)
valor2 = float(valor2)
print("VALOR A PAGAR: R$ {:.2f}".format((valor * un) + (valor2 * un2)))

File: Cálculo_Simples.js
const fs = require("fs");
let r = fs.readFileSync(0,"utf8");

let input= r.split("\n");
let line1 = input[0].split(" ");
let c1 = parseInt(line1[0]);
let q1 = parseInt(line1[1]);
let v1 = parseFloat(line1[2]);
let line2 = input[1].split(" ");
let c2 = parseInt(line2[0]);
let q2 = parseInt(line2[1]);
let v2 = parseFloat(line2[2]);

let ans  = (q1*v1)+(q2*v2);
console.log("VALOR A PAGAR: R$ "+ans.toFixed(2));